app/queries/find_exact_query.rb

```diff
@@ -38,6 +38,6 @@ def base_scope
   end
 
   def find_exact(query_field, query)
-    FindExactMatch.new(query_field: query_field, scope: base_scope).call(query).first
+    FindExactMatch.new(query_field: query_field, scope: base_scope).call(query)
```

Comment:
Does FindExactMatch still make sense as an individual class once changed?

Reply:
I think so, though they're very tightly coupled, this change didn't change the relationship between the classes